test/ffmpeg-vaapi/encode/10bit/hevc.py

```diff
@@ -26,12 +26,10 @@ def get_vaapi_profile(self):
     }[self.profile]
 
 spec = load_test_spec("hevc", "encode", "10bit")
+spec_r2r = load_test_spec("hevc", "encode", "10bit", "r2r")
 
 class cqp(HEVC10EncoderTest):
-  @platform_tags(HEVC_ENCODE_10BIT_PLATFORMS)
-  @slash.requires(have_ffmpeg_hevc_vaapi_encode)
-  @slash.parametrize(*gen_hevc_cqp_parameters(spec, ['main10']))
-  def test(self, case, gop, slices, bframes, qp, quality, profile):
+  def init(self, spec, case, gop, slices, bframes, qp, quality, profile):
```

Comment:
probably better not to use the same variable name as the global `spec` variable.

Reply:
changed to tspec to avoid confusion with global name